plugins/Sf2Player/Sf2Player.cpp

```diff
@@ -537,6 +545,38 @@ void  Sf2Instrument::updateChorus()
 
 
 
+void Sf2Instrument::updateTuning()
+{
+	if (instrumentTrack()->microtuner()->enabledModel()->value())
+	{
+		auto centArray = std::array<double, 128>{};
+		double lowestHz = powf(2.f, -69.f / 12.f) * 440.f;// Frequency of MIDI note 0, which is approximately 8.175798916 Hz
+		for (int i = 0; i < 128; ++i)
+		{
+			// Get desired Hz of note
+			centArray[i] = instrumentTrack()->microtuner()->keyToFreq(i, DefaultBaseKey);
+			// Convert Hz to cents
+			centArray[i] = 1200.f * log2(centArray[i] / lowestHz);
+		}
+		
+		fluid_synth_activate_key_tuning(m_synth, 0, 0, "", centArray.data(), true);
```

Comment:
These tuning functions were introduced in FluidSynth 1.1.0, so the minimum required version in the root `CMakeLists.txt` should be bumped.